pkg/streamingpromql/operators/aggregations/aggregation.go

```diff
@@ -263,7 +277,17 @@ func (a *Aggregation) emitAnnotation(generator types.AnnotationGenerator) {
 	a.Annotations.Add(generator(metricName, a.Inner.ExpressionPosition()))
 }
 
+func (a *Aggregation) emitAnnotationParam(param float64, generator types.AnnotationParamGenerator) {
```

Comment:
[nit] `emitParamAnnotation` reads nicer to me

(similar feedback applies for `ParamAnnotationGenerator` rather than `AnnotationParamGenerator`)